devel-tools/vignetting/README.md

```diff
@@ -14,10 +15,12 @@ Instructions for generating vignetting spline fits.
    photons per star.  Currently, `magnorm=22.25` will produce ~100k
    incident photons per star.
 
-4. To run the simulation run `galsim imsim-vignetting.yaml`.
+4. To run the simulation run `galsim imsim-vignetting.yaml`.  For
+ComCam, you can add `eval_variables.bisComCam=True`.
```

Comment:
What does adding this do?

Reply:
It sets [this](https://github.com/LSSTDESC/imSim/blob/comcam_vignetting/devel-tools/vignetting/imsim-vignetting.yaml#L17) eval variable, which is then used in the yaml file to select between comcam/lsstcam settings in a few places.  E.g., [here](https://github.com/LSSTDESC/imSim/blob/comcam_vignetting/devel-tools/vignetting/imsim-vignetting.yaml#L30).  

The `b` in `bisComCam` (for boolean) is unfortunate.  (Would have been nice to just have `isComCam` which is more readable.). But that's a limitation of the galsim config system.  I can add a bit more text to the README.

Reply:
Maybe `bIsComCam` or `bUseComCam` is better?

Reply:
Okay, I updated the README text and went with `bIsComCam`.